rfcs/accordion-component.md

```diff
@@ -0,0 +1,124 @@
+# Accordion Component
+
+## Usage
+
+```js
+import {Accordion, AccordionItem, Root, Toggle} from './index';
+
+export default () => {
+  return (
+    <div>
+      <Accordion>
+        <AccordionItem key={1} title={'First title'}><div>First content</div></AccordionItem>
+        <AccordionItem key={2} title={'Second title'}><div>Second content</div></AccordionItem>
+      </Accordion>
+    </div>
+  );
+};
+````
+
+```js
+import {Accordion, AccordionItem, Root, Toggle} from './index';
+import {styled} from '../styles/index';
+
+const CustomRoot = styled(Root, props => ({
+  textColor: 'red',
+}));
+
+const CustomToggle = styled(Toggle, props => ({
+  backgroundColor: props.$isExpanded ? 'green' : 'blue',
+}));
+
+export default () => {
+  return (
+    <div>
+      <Accordion
+        onToggle={(event, index, isExpanded) => {console.log('Accordian ' + index + ' is ' + (isExpanded ? 'expanded' : 'collapsed') )}}
+        overrides={{
+          Root: CustomRoot,
+          Toggle: CustomToggle,
+        }}
+        expanded={[false, true]}
+      >
+          <AccordionItem key={1} title={'First title'}><div>First content</div></AccordionItem>
+          <AccordionItem key={2} title={'Second title'}><div>Second content</div></AccordionItem>
+          <AccordionItem key={3} title={'Third title'}><div>Third content</div></AccordionItem>
+      </Accordion>
+    </div>
+  );
+};
+```
+
+## Exports
+
+* `Accordion`
+* `StatefulAccordion`
+* `StatefulAccordionContainer`
+* `AccordionItem`
+* `StyledRoot`
+* `StyledToggle`
+* `StyledTitle`
+* `StyledContent`
+
+## `Accordion` API
+
+* `children: Array<AccordionItem>` - Required
+  All accordion items in component. Each `AccordionItem` has:
+  * `title: React$Node` - title of accordion item
+  * `children: React$Node` - content of accordion item is represented as children of it correspondingly
+* `expanded: Array<boolean>` - Optional
```

Comment:
I think this prop should be on every accordion item, maybe in addition to the top level prop.

Reply:
ok